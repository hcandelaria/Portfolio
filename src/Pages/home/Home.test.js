import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '.';

describe('Home page', () => {
  it('should mount', () => {
    render(<Home />);
    const homePage = screen.getByTestId('homePage');
    expect(homePage).toBeInTheDocument();
  });

  it('should render welcome text', () => {
    render(<Home />);
    const welcomeComponent = screen.getByText(/welcome/i);
    expect(welcomeComponent).toBeInTheDocument();
  });

  it('should render correctly', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot();
  })

});
