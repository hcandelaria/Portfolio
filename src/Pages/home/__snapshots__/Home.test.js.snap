// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render correctly 1`] = `
<div
  className="d-flex align-items-center justify-content-center col-md-10"
  data-testid="homePage"
  id="homePage"
>
  <div
    data-testid="welcome"
    id="welcome"
  >
    Welcome
  </div>
</div>
`;
